Documents/idea document 2/ideadocument2.md

```diff
@@ -13,7 +13,33 @@ With these two sources data which is relevant for the application, can be collec
 To derive this information after the collection, data must be analyzed, which will be discussed in the Section Methods.
 
 # Methods
-(Thom)
+This chapter specifies which methods will be used for the neighbourhood classification. The several steps that have to be taken to extract useful information out of the data are dispayed in a pipeline diagram, after which the steps are elaborated somewhat more.
+
+### Overall system architecture
+The following graph shows how input data (e.g. tweets) are processed into neighbourhood classifications.
+<p align="center">
+  <img alt="Workbench electron-browser component" src="../images/pipeline.png" width="60%"><br/>
+  <i>Graphical representation of the proposed system architecture</i>   
+</p>
```

Comment:
That will be changed at the end, but it is easier to search for "figure-thom-pipeline" and replace all to "X" than scrolling through the text